docs/source/algorithms/SingleCrystalDiffuseReduction-v1.rst

```diff
@@ -67,6 +67,23 @@ For example setting SymmetryOps to "P 31 2 1", "152" or "x,y,z;
 -y,x-y,z+1/3; -x+y,-x,z+2/3; y,x,-z; x-y,-y,-z+2/3; -x,-x+y,-z+1/3"
 are equivalent.
 
+Temporary Workspaces
+####################
+
+If the KeepTemporaryWorkspaces option is True the data and the
+normalization in addition to the nomalized data will be
+outputted. This allows you to run separate instances of
+SingleCrystalDiffuseReduction and combine the results. They will have
+names "OutputWorkspace" + '_data' and "OutputWorkspace" +
```

Comment:
I would say here that if OutputWorkspace is ws, then the temporary workspace names will be ws_data, ... Otherwise the explanation is too complicated. Plus you are using different quotation marks